cloud-init-docker/cloud-init/apps/data-connect-trino/cloud-init

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# This client's runtime credentials within the cluster - allows it to call out to other services
+createWalletServiceClient "${APP_NAME}" "OAuth Client representing GA4GH Data Connect Service" "app_wallet_client-credential"
```

Comment:
If you change the cloud-init yml's `WALLET_URL` to internal route as I commented in that file, you'll need to make changes to these scripts to specify wallet url and resource. Here's how I did it in collection-service: https://github.com/DNAstack/collection-service/blob/main/cloud-init-docker/cloud-init/apps/collection-service/cloud-init#L9